Makefile

```diff
@@ -167,8 +167,8 @@ endif
 proto-gen: check-proto-deps
 	@echo "Generating Protobuf files"
 	@go run github.com/bufbuild/buf/cmd/buf generate
-	@mv ./proto/tendermint/abci/types.pb.go ./abci/types/
-	@cp ./proto/tendermint/rpc/grpc/types.pb.go ./rpc/grpc
```

Comment:
No idea why this generated Go file had to be copied while the one for `abci` above is moved. @julienbrt @thanethomson can you clarify why this duplication was needed in https://github.com/tendermint/tendermint/pull/9356?

Reply:
This is a legacy thing, it seems as though it was deemed best to have the generated ABCI proto code colocated with the rest of the ABCI code.

Similarly with the generated gRPC code.

The better approach is to have it all generated into an `/api/` package in the root of the project, as per Buf's guidelines.